foodsaving/invitations/api.py

```diff
@@ -40,6 +47,19 @@ def get_throttles(self):
             self.throttle_classes = (InvitesPerDayThrottle, )
         return super().get_throttles()
 
+    @action(detail=True, methods=['POST'], permission_classes=(IsAuthenticated, CanResend))
+    def resend(self, request, **kwargs):
+        """
+        Resend invitation email
+        """
+        self.check_permissions(request)
+        instance = self.get_object()
+        self.check_object_permissions(request, instance)
+
+        instance.resend_invitation_email()
+
+        return Response(instance.created_at)
```

Comment:
I would either
- return nothing as it's not being used in the frontend
- or return the serialized invitation object, to have same type of response as the "create" action

You can do the latter by accessing the pre-defined invitation serializer:
```py
serializer = self.get_serializer(instance)
return Response(serializer.data)
```

Reply:
I think it's best to return nothing. I did that return by mistake, sorry.